components/molecules/ContributorListTableHeader/contributor-list-table-header.tsx

```diff
@@ -7,9 +7,14 @@ import Checkbox from "components/atoms/Checkbox/checkbox";
 interface ContributorListTableHeadersProps {
   selected?: boolean;
   handleOnSelectAllContributor?: (contributor: any) => void;
+  range?: number;
 }
 
-const ContributorListTableHeaders = ({ selected, handleOnSelectAllContributor }: ContributorListTableHeadersProps) => {
+const ContributorListTableHeaders = ({
+  selected,
+  handleOnSelectAllContributor,
+  range = 30,
```

Comment:
Range is passed in now so when you filter, the graph associated with that column can be updated to reflect the correct range filter.